cloud-init/copr_build.py

```diff
@@ -142,6 +141,16 @@ def main(srpm, copr_conf=DEFAULT_COPR_CONF, dev=None):
                            help='use cloud-init-dev instead of cloud-init')
     ARGPARSER.add_argument('-s', '--srpm', dest='srpm', action='store',
                            required=True, help='srpm file')
+    ARGPARSER.add_argument('-p', '--project-id', action='store', type=int,
+                           required=True, help='COPR project id. Overrides -d/--dev.')
```

Comment:
If  --project is required, then what's --dev for?

Reply:
heh, it should not be required, thanks for spotting it.